### DISCUSSION TITLE
On-screen overlay settings not working

### DISCUSSION THREAD

**Discussion by miguelinksy** [eduardofilo/RG350_adam_image#106] [RetroArch topics]
Hey! I have been trying to come up with a custom menu for the RG280v bundled with overlays for all handheld consoles for a  better pixel perfect experience. (The way scaling works on this unit is quite confusing and very different to the way upstream Retroarch works) The Adam image has been great so far, but whenever I try to activate the on screen overlays I always get an error displaying "Directory not found" which I can't seem to get around. Any ideas on how to activate this feature on RA? Is this known problem?

**Comment by eduardofilo** [comment]
Hi.
I am not an expert, but it seems that overlay in the RetroArch version for OpenDingux it is not enabled. In ArkOS on RG351, I can see the directory configuration for Overlays, but in RG280 it isn't there. Surely the `On-Screen Overlay` menu entry should not be displayed.

![IMG_20211027_223604](https://user-images.githubusercontent.com/2923728/139143409-739ca519-95cc-4571-859e-fec67be7e724.jpg)
